TheJumpCloud/support
# Get-JC Policy Functions Branch

## Related Card/Ticket:
https://jumpcloud.atlassian.net/browse/SA-9

## What does this solve/add?
Adds in the following Policy related functions to the module:
* Get-JCPolicy
* Get-JCPolicyTargetGroup
* Get-JCPolicyTargetSystem
* Get-JCPolicyResult

* Get-Hash_PolicyID_Name
* Get-Hash_PolicyName_ID


## Is there anything particularly tricky?
- No

## How should this be tested?
- Run bespoke pester tests added into "/tests/JumpCloud.1.10.0.Tests.ps1"
- Verify function behavior
